lib/cadet/jobs/autograder/lambda_worker.ex

```diff
@@ -18,16 +18,20 @@ defmodule Cadet.Autograder.LambdaWorker do
   storage job.
   """
   def perform(params = %{answer: answer = %Answer{}, question: %Question{}}) do
-    lambda_params = build_request_params(params)
+    with lambda_params <- build_request_params(params),
+         {:hasTestcases?, true} <- {:hasTestcases?, !Enum.empty?(lambda_params.testcases)} do
```

Comment:
why suddenly camel case here? `contains_testcases?` seems like a better fit haha

Reply:
Also, i think an if else or a case here will be cleaner leh the function is quite unreadable! With should be used for multi step transaction! defo prefer if else of cond if youre just checking one thing!

Reply:
Thanks for the suggestion